### DISCUSSION TITLE
Printer not respecting bed temp set in Orca Slicer

### DISCUSSION THREAD

**Discussion by Avpman2** [xblax/flashforge_ad5m_klipper_mod#170] [Q&A] [answered]
Admittedly, I do not know if this is an Orcaslicer or Klipper Mod issue. When I override the bed temperature preset the printer is not updated. For example, I set the bed temp to 80deg but in Klipper it still shows 60deg. See screenshots. How do I diagnose where the issue is?

![orc2](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/3275462/463bc2ba-c80c-4ae2-b0f5-65b93e006b44)
![orca1](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/3275462/23aec3a4-6517-464f-be36-2211133e609e)

**Comment by consp** [comment]
Could you show the generated gcode (at least the part setting the bed temperature)? That might help. No need to upload the entire file, just the setup part setting the temp.

**Reply by Avpman2** [reply]
Sure, from what little I know, it looks like the temperature in the Gcode is correct. 
M190 S70
M104 S220


![Screenshot 2024-05-04 165016](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/3275462/130ce298-d60b-4e8e-b26f-e51abb6bbcb0)

**Reply by consp** [reply]
Interesting, never had those issues. Not my field of expertese though, @xblax any idea what this might be?

**Reply by xblax** [reply]
I'm pretty sure the start G-Code is not correct and it just sets the default of the START_PRINT macro instead.

**Comment by Tiwatz** [comment]
What is your Machine start gcode in Orca?

**Reply by Avpman2** [reply]
START_PRINT
M190 S[bed_temperature_initial_layer_single]
M104 S[nozzle_temperature_initial_layer]
G90
M83
G1 Z5 F6000
G1 E-1.5 F800
G1 X110 Y-110 F6000
G1 E2 F800
G1 Y-110 X55 Z0.25 F4800
G1 X-55 E8 F2400
G1 Y-109.6 F2400
G1 X55 E5 F2400
G1 Y-110 X55 Z0.45 F4800
G1 X-55 E8 F2400
G1 Y-109.6 F2400
G1 X55 E5 F2400
G92 E0

**Reply by Tiwatz** [reply]
Have a look here: https://github.com/xblax/flashforge_ad5m_klipper_mod/discussions/79#discussioncomment-8826586

**Reply by Avpman2** [reply]
Just so I am clear, I should delete everything except what's posted in #79 Orca Confusion?

**Reply by Tiwatz** [reply]
Yes. Your start g-code should look like this `START_PRINT BED_TEMP=[bed_temperature_initial_layer_single] EXTRUDER_TEMP=[nozzle_temperature_initial_layer]`
And the end g-code should be `END_PRINT`

**Reply by Avpman2** [reply]
Excellent, thank you!
Is there a place in Orca where I can set the default start/end code (other than editing the printer settings for each nozzle/filament type? I have Googled to no avail.

**Comment by Avpman2** [comment, marked as answer]
I found the "problem." The FW sets the extruder and bed temperatures to defaults of 200c/60c, respectively, during Auto Bed Leveling. Once the leveling has been completed, it sets the temperatures to whatever OrcaSlicer is set to. I discovered this by accident while looking around the Guppyscreen, which specifically reports that those temperatures are being used during leveling.
